cloudbrain/rt_server/rt_server.py

```diff
@@ -118,6 +129,10 @@ def on_close(self):
     def send_heartbeat(self):
         self.broadcast(self.clients, 'message')
 
+    def metric_exists(self, device_id, device_name, metric):
+        return (self.subscribers.has_key(device_id)
+                and self.subscribers[device_id].has_key(device_name)
+                and self.subscribers[device_id][device_name].has_key(metric))
 
 class MockHandler(RequestHandler):
```

Comment:
This mocked class is a good thing to take out and start writing some proper unit tests :smile: